wiki/en/Installation-for-Windows.md

```diff
@@ -73,6 +77,15 @@ Have a look at [this video](https://youtu.be/_GzOsitVgLI) by [trombonepizza](htt
 
 Official and further information about how to configure ASIO4ALL is documented in the official [ASIO4ALL FAQs on the ASIO4ALL website](https://www.asio4all.org/index.php/help/faq/){: target="_blank" rel="noopener noreferrer"}.
 
+## JACK
+
+Similar to the Linux version, Jamulus can also use [JACK](https://jackaudio.org/faq/jack_on_windows.html) on Windows. To get started with the JACK version:
+
+1. **Download and install JACK for Windows**. Get it [from the jackaudio page](https://jackaudio.org/downloads/).
+1. [Download Jamulus (JACK version)]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }}){: .button}
+1. Now install Jamulus similar to the Windows ASIO version as mentioned above.
+1. Afterwards, to set up JACK, follow the [Linux setup guide](Installation-for-Linux).
+
```

Comment:
We could also move this somewhere else too, but I think it's better to put it at the bottom.

Reply:
Is there anything we can say about either the JACK or ASIO versions to help someone who might not know which to choose? Could we say ASIO is the preferred version but that JACK may allow more advanced use in some cases? I'm not too sure about the differences myself.

Reply:
If the user's soundcard ASIO driver is up to scratch and the only ASIO app they use is Jamulus, then there's no reason to use anything but the ASIO version of Jamulus.

If they use other ASIO applications, they have a number of choices: Jack for Windows supports I/O using ASIO, which allows it to be used with other ASIO application without JACK support.  So you could choose to use the ASIO version of Jamulus with JACK anyway.  Or you could use the JACK-native version of Jamulus, which, if you're going to use JACK at all, probably makes more sense.

If the user's soundcard ASIO driver _isn't_ up to scratch, they might find JACK a better option than ASIO4ALL.  There's probably still more help available on the Web regarding ASIO4ALL than JACK for Windows, of course...  So, unless the user is reasonably happy working out how to set up something like ASIO4ALL or JACK for Windows, ASIO4ALL is likely the better choice (their problems will find more help in fixing them).  Where they're happy trying anything, JACK might open up opportunities they'd not thought to try, though.

In simple terms: if you're likely to want to link multiple applications together, JACK _might_ be a good alternative.

Reply:
> if you're likely to want to link multiple applications together, JACK might be a good alternative.

So can we assume 99% of all Windows users have never heard of JACK and don't need to know about it? And the remaining 1% probably know about as part of some specialised thing they're doing, independent of their use of Jamulus. 

In that case, I'd just have a straight download link to the app and cut out the JACK download steps and stuff because that's going to be redundant and draws us into supporting JACK issues. 

Instead, I'd just have a line under the upgrading statement like this:

"If you use [JACK](https://jackaudio.org/) on Windows, we have a version that [supports JACK](link)"

and then the rest of the page is about ASIO. Otherwise it just clogs up the page with stuff that virtually nobody is going to want to know.

Reply:
Yes. Probably. However I know some sites proposing that JACK works better than ASIO4ALL due to Port audio support. But I can't verify that JACK works better.

Reply:
Be that as it may, the JACK version is only really relevant if you know about JACK already. Inviting people to "get started" with JACK will get us into trouble if people think its some easy alternative to all the ASIO stuff they need to read though on that page. I'd just put it in as a link and be done with it.

Reply:
@gilgongo changed.